### DISCUSSION TITLE
Property 'OPT' does not exist on type 'CompoundedComponent'.

### DISCUSSION THREAD

**Discussion by Ladvace** [ant-design/ant-design#48275] [Q&A] [answered]
I was trying to use the OTP component but I'm getting this error as OTP does not exist

`Property 'OPT' does not exist on type 'CompoundedComponent'.`

<img width="592" alt="image" src="https://github.com/ant-design/ant-design/assets/31326058/150c3185-5824-4031-a38c-93c3152fa76f">

**Comment by li-jia-nan** [comment]
Can you provide a reproduction ?

**Comment by afc163** [comment, marked as answer]
```diff
- Input.OPT
+ Input.OTP
```

**Reply by Ladvace** [reply]
Oh lol, sorry, my bad

**Comment by yogithesymbian** [comment]
its only on version `5.20.2` ? how about if we use version `5.15.3` ?

**Reply by yogithesymbian** [reply]
> its only on version `5.20.2` ? how about if we use version `5.15.3` ?

because i get error on lint type at `OTP` 

```Property 'OTP' does not exist on type 'CompoundedComponent'.```
no test yet

**Comment by Shabi-x** [comment]
I had a similar problem. How to solve it
